src/elements/ProgressBar/ProgressBar.styl

```diff
@@ -62,6 +62,11 @@
 
   sizes('font-size')
 
+  height $height-control
+  width $width-control
+  box-sizing $box-sizing-control
+
+
```

Comment:
The separator between two blocks must be one newline only.

Reply:
```suggestion
height $height-control
width $width-control
box-sizing $box-sizing-control

```
// Children

Reply:
What I meant was that only one blank line should be left here.